ledger/participant-integration-api/src/main/scala/platform/store/dao/LedgerDao.scala

```diff
@@ -79,6 +79,13 @@ private[platform] trait LedgerDaoTransactionsReader {
       filter: FilterRelation,
       verbose: Boolean,
   )(implicit loggingContext: LoggingContext): Source[GetActiveContractsResponse, NotUsed]
+
+  def getContractStateEvents(
```

Comment:
A comment with a short description would be nice here

Reply:
Done.